packages/web-components/src/components/masthead/cloud/cloud-masthead-composite.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement } from 'lit-element';
+import ifNonNull from 'carbon-web-components/es/globals/directives/if-non-null.js';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import Cookies from 'js-cookie';
+import '../masthead';
+import '../masthead-logo';
+import '../masthead-l1';
+import '../masthead-l1-name';
+import '../masthead-menu-button';
+import '../masthead-global-bar';
+import './cloud-masthead-profile';
+import '../masthead-profile-item';
+import '../megamenu';
+import '../megamenu-top-nav-menu';
+import '../megamenu-left-navigation';
+import '../megamenu-category-link';
+import '../megamenu-category-group';
+import '../megamenu-category-group-copy';
+import '../megamenu-link-with-icon';
+import '../megamenu-overlay';
+import '../top-nav';
+import '../top-nav-l1';
+import '../top-nav-name';
+import '../top-nav-item';
+import '../top-nav-menu';
+import '../top-nav-menu-item';
+import '../left-nav';
+import '../left-nav-name';
+import '../left-nav-item';
+import '../left-nav-menu';
+import '../left-nav-menu-item';
+import '../left-nav-menu-category-heading';
+import '../left-nav-overlay';
+import '../masthead-search-composite';
```

Comment:
We can remove all the component imports here except for `import './cloud-masthead-profile';` since we're extending from `dds-masthead-composite`